### PR TITLE
Fix multiple scalar issues:

### PR DESCRIPTION
**Pull request by nickyfantasy**
-download link / outliers should take boolean value instead of array
-download link now show proper selector of runsItem and download button
-Use tag as key to prevent chart re-rendering
-Add missing label in scalar config and fix UI

### CODE REVIEW

File: frontend/src/histogram/ui/ChartPage.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
             <div class="visual-dl-chart-page-box">
                 <ui-chart
                     v-for="tagInfo in filteredPageList"
-                    :key="tagInfo.run + tagInfo.tag.displayName"
+                    :key="tagInfo.tag"
```
Comment (jetfuel) [inline frontend/src/histogram/ui/ChartPage.vue]: We should avoid using non-primitive value as the key. Vue will complain this. 
Let's just use Varun's index suggestion.

File: frontend/src/images/ui/ChartPage.vue [hunk 2]
```diff
@@ -4,7 +4,7 @@
             <ui-image
                 class="visual-dl-chart-image"
                 v-for="tagInfo in filteredPageList"
-                :key="tagInfo.run + tagInfo.tag.displayName"
+                :key="tagInfo.tag"
```
Comment (jetfuel) [inline frontend/src/images/ui/ChartPage.vue]: We should avoid using non-primitive value as the key.

File: frontend/src/scalars/ui/ChartPage.vue [hunk 3]
```diff
@@ -4,8 +4,8 @@
             <div ref="chartPageBox" class="visual-dl-chart-page-box">
                 <ui-chart
                     v-for="tagInfo in filteredTagList"
+                    :key="tagInfo.tag"
```
Comment (jetfuel) [inline frontend/src/scalars/ui/ChartPage.vue]: We should avoid using non-primitive value as the key.

File: frontend/src/scalars/ui/Config.vue [hunk 4]
```diff
@@ -18,31 +24,32 @@
             <v-radio label="Wall" value="wall"></v-radio>
         </v-radio-group>
 
-        <label class="label">Tooltip sorting method</label>
```
Comment (jetfuel) [inline frontend/src/scalars/ui/Config.vue:21]: We need to display this Label.

Comment (nickyfantasy) [reply frontend/src/scalars/ui/Config.vue:21]: v-selector has this label embedded

**Review by jetfuel — APPROVED** [review]

File: frontend/src/scalars/ui/Config.vue (hunk 4 above)
Comment (jetfuel) [reply frontend/src/scalars/ui/Config.vue:21]: I think we need to null the v-model in order for the label to show up then.

**Pull request merged**